src/main/java/monke/tasks/Task.java

```diff
@@ -0,0 +1,67 @@
+package monke.tasks;
+
+/**
+ * The Task class represents a task in the Monke application.
+ * It is an abstract class and serves as the base class for specific task types.
+ */
+public abstract class Task {
```

Comment:
Good naming of classes as nouns and functions as verbs. This is seen throughout the code. 👍